docs/terms/evaluation_parameter.md

```diff
@@ -1,15 +1,132 @@
+
+To do so, you would define the parameter in question (`upstream_row_count`) by assigning it to the `$PARAMETER` value in a dictionary.  Then, you would provide the temporary value for that parameter by setting it as the value of the `$PARAMETER.<parameter_in_question>` key in the same dictionary.  Or, in this case, the `$PARAMETER.upstream_row_count`.
+
+For an example of this, see below:
+
+```python title="Python code"
+my_df.expect_table_row_count_to_equal(
+    value={"$PARAMETER": "upstream_row_count", "$PARAMETER.upstream_row_count": 10},
+    result_format={'result_format': 'BOOLEAN_ONLY'}
+)
+```
+
+This will return `{'success': True}`.
+
+An alternative method of defining the temporary value for an Evaluation Parameter is the `set_evaluation_parameter()` method, as shown below:
+
+```python title="Python code"
+my_df.set_evaluation_parameter("upstream_row_count", 10)
+```
+
+If the Evaluation Parameter's value is set in this way, you do not need to set it again (or define it alongside the use of the `$PARAMETER` key) for future Expectations.
+
+More typically, when validating Expectations, you can provide Evaluation Parameters that are only available at runtime.
+
+```python title="Python code"
+my_df.validate(
+    expectation_suite=my_dag_step_config, 
+    evaluation_parameters={"upstream_row_count": upstream_row_count}
+)
+```
+
+<ValidateHeader/>
+
+Evaluation Parameters that are configured as part of a Checkpoint's Expectations will be used without further interaction from you.  Additionally, Evaluation Parameters will be stored by having the `StoreEvaluationParametersAction` subclass of the `ValidationAction` class defined in a Checkpoint configuration's `action_list`.
+
+
+
+## Features
+
+### Dynamic values
+
+Evaluation Parameters are defined by expressions that are evaluated at run time and replaced with the corresponding values.  These expressions can include such things as:
+- Values from previous Validation runs, such as the number of rows in a previous Validation.
+- Values modified by basic arithmatic, such as a percentage of rows in a previous Validation.
+- Temporal values, such as "now" or "timedelta."
+- Complex values, such as lists.
+
+:::note
+Although complex values like lists can be used as the value of an Evaluation Parameter, you cannot currently combine complex values with arithmetic expressions.
+:::
+
+## API basics
+
+### How to create
+
+An Evaluation Parameter is "created" when a parameter is referenced in the results of another Expectation Suite's Validation.  To store them, define a `StoreEvaluationParametersAction` subclass of the `ValidationAction` class in a Checkpoint configuration's `action_list`, and run that Checkpoint.
+
+Evaluation Parameters are *referenced* by making a dictionary with the `$PARAMETER` key.  The value for this key will be the Evaluation Parameter's expression, which will be evaluated at run time and replaced with the value described by said expression.
```

Comment:
I think we should include URNs here as well

Reply:
I've added a reference to them here as well (and done a bit of an overhaul of this section for clarity's sake).